pkg/skaffold/deploy/helm.go

```diff
@@ -220,9 +220,9 @@ func (h *HelmDeployer) Render(context.Context, io.Writer, []build.Artifact, []La
 
 // exec executes the helm command, writing combined stdout/stderr to the provided writer
 func (h *HelmDeployer) exec(ctx context.Context, out io.Writer, useSecrets bool, args ...string) error {
+	args = append(args, h.Flags.Global...)
```

Comment:
@ryanhagen please add a another test case where flags is not empty here https://github.com/GoogleContainerTools/skaffold/blob/49888e0abdd6249c18fdb311970a1d7734f48b1a/pkg/skaffold/deploy/helm_test.go#L734

Reply:
See below commit